demos/yielding/README.md

```diff
@@ -0,0 +1,112 @@
+# Yielding
+
+In this demo, we try to show the usage of yielding, which is one characteristic of a multitasking system. 
```

Comment:
"In this demo we show the usage of yielding, which is one element of a multitasking system.

- remove the first comma
- we don't try we definitely do :) (whether the reader feels we succeeded is up to them)
- characteristic -> element. Characteristic seems like an adjective thing to me, like "suqishy" or "bold". Ok maybe not in this context but you get the idea.